styles.css

```diff
@@ -110,11 +114,13 @@ img {
 
 .no-bullets {
 list-style-type: none; 
+margin: 4px 10px;
 }
 
 .buttontext {
-  background-color: rgba(176, 146, 201, 0.7);
+  background-color: rgba(164, 130, 192, 0.7);
   padding: 1px 10px;
+  margin: 5px;
```

Comment:
You could add a box-shadow to your buttons.

Reply:
Thank you, good advise to make it more pretty :-)